source/reporting/index.html.md.erb

```diff
@@ -233,6 +233,17 @@ These take inputs based on a subset of [the ISO
 8601 standard](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard).
 For example, a valid input would be `2015-08-13T12:35:00Z`.
 
+#### Filter by when your PSP sent payments to your bank account
+
+If your PSP is Stripe, you can use the following query parameters to filter by when your PSP sent payments to your bank account:
```

Comment:
need to clarify how nulls a treated

if the payout hasn't happened yet is the payment included or excluded from the result set?

if I wanted to list all payments which haven't been settled, could I do that?

Reply:
That's a great point, filtering on created date doesn't run into this complexity as transactions are guaranteed to be created.

I'm not sure how it should be communicated but the answer to the functionality would likely be: 

> If the payout hasn't happened yet is the payment included or excluded from the result set?
Excluded, this wouldn't meet the search criteria

> If I wanted to list all payments which haven't been settled, could I do that?
No